modules/farm_fd2_examples/src/entrypoints/selector_base/App.vue

```diff
@@ -53,14 +53,26 @@
         </td>
       </tr>
       <tr>
-        <td>Include Add Button</td>
+        <td>PopupUrl</td>
```

Comment:
Seems like only one or the other of these two buttons should be enabled.  Alternatively you could use a switch to add a non-`null` Url instead of two buttons.